webui/react/src/components/CompareHyperparameters.test.mock.tsx

```diff
@@ -0,0 +1,412 @@
+import React from 'react';
+
+import { Scale } from 'types';
+
+import CompareHyperparameters from './CompareHyperparameters';
+export const METRIC_DATA = {
```

Comment:
for the mock data, are there api-ts-sdk client types that can be associated with it? For example if there was a `MetricData` type exported from `api-ts-sdk` then you can do `export const METRIC_DATA: MetricData = {` and if the shape changes in the future this test would capture the change.

If `MetricData` is not available and `METRIC_DATA` is actually a form of a processed data, it might be better to mock the unprocessed data with the corresponding typing and then converting it to `METRIC_DATA` to capture the benefit mentioned above.

Reply:
not from `api-ts-sdk` but adding typing with `TrialMetricData` from `useTrialMetrics`